subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/ArtifactHandlerScope.kt

```diff
@@ -41,6 +43,9 @@ private constructor(
             ArtifactHandlerScope(artifacts)
     }
 
+    override val delegate: ArtifactHandler
```

Comment:
:question: shouldn't those be `internal`?

Reply:
`override` takes care of it.